### PR TITLE
Sync from main && Add two-way ssl support for tensorflow-serving-cluster

### PR DESCRIPTION
**Pull request by BuJianlin**

### CODE REVIEW

File: documents/readthedoc/docs/source/Solutions/tensorflow-serving-cluster/index.rst [hunk 1]
```diff
@@ -181,29 +181,40 @@ The converted model file will be under::
...
 
-1.2 Create the TLS certificate
+1.2 Create the SSL/TLS certificate
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-We choose gRPC TLS and create the one-way TLS Keys and certificates by setting
+We choose gRPC SSL/TLS and create the SSL/TLS Keys and certificates by setting
 TensorFlow Serving domain name to establish a communication link between client
 and TensorFlow Serving.
 
+For ensuring security of the data being transferred between a client and server, SSL/TLS can be implemented either one-way or two-way.
```
Comment (RodgerZhu) [inline documents/readthedoc/docs/source/Solutions/tensorflow-serving-cluster/index.rst:190]: SSL/TLS can be implemented either one-way authentication or two-way authentication.

File: documents/readthedoc/docs/source/Solutions/tensorflow-serving-cluster/index.rst [hunk 2]
```diff
@@ -181,29 +181,40 @@ The converted model file will be under::
...
       ./generate_oneway_ssl_config.sh ${service_domain_name}
       tar -cvf ssl_configure.tar ssl_configure
 
-   two-way::
+      ``generate_oneway_ssl_config.sh`` will generate the directory 
+      ``ssl_configure`` which includes ``server/*.pem`` and ``ssl.cfg``.
+      ``server/cert.pem`` will be used by the remote client and ``ssl.cfg`` 
+      will be used by TensorFlow Serving.
+
+   two-way(server and client verify each other)::
```
Comment (RodgerZhu) [inline documents/readthedoc/docs/source/Solutions/tensorflow-serving-cluster/index.rst:205]: two-way TLS authentication

File: documents/readthedoc/docs/source/Solutions/tensorflow-serving-cluster/index.rst [hunk 3]
```diff
@@ -181,29 +181,40 @@ The converted model file will be under::
...
+We choose gRPC SSL/TLS and create the SSL/TLS Keys and certificates by setting
 TensorFlow Serving domain name to establish a communication link between client
 and TensorFlow Serving.
 
+For ensuring security of the data being transferred between a client and server, SSL/TLS can be implemented either one-way or two-way. 
+
 For example::
 
-   one-way::
+   one-way(client verify server)::
```
Comment (RodgerZhu) [inline documents/readthedoc/docs/source/Solutions/tensorflow-serving-cluster/index.rst:194]: one-way TLS authentication(client verifies server)

File: documents/readthedoc/docs/source/Solutions/tensorflow-serving-cluster/index.rst [hunk 4]
```diff
@@ -413,7 +420,13 @@ request client is good.
 ^^^^^^^^^^^^^^^^^^^^^^^
 Start the remote request with dummy image::
 
-   python3 ./resnet_client_grpc.py --url ${service_domain_name}:8500 --crt `pwd -P`/ssl_configure/server.crt --batch 1 --cnum 1 --loop 50
+   one-way::
```
Comment (RodgerZhu) [inline documents/readthedoc/docs/source/Solutions/tensorflow-serving-cluster/index.rst:434]: one-way TLS authentication::

File: documents/readthedoc/docs/source/Solutions/tensorflow-serving-cluster/index.rst [hunk 5]
```diff
@@ -413,7 +420,13 @@ request client is good.
...
+
+      python3 ./resnet_client_grpc.py --batch 1 --cnum 1 --loop 50 --url ${service_domain_name}:8500 --crt `pwd -P`/ssl_configure/server/cert.pem
+
+   two-way::
```
Comment (RodgerZhu) [inline documents/readthedoc/docs/source/Solutions/tensorflow-serving-cluster/index.rst:438]: two-way TLS authentication::

File: documents/readthedoc/docs/source/Solutions/tensorflow-serving-cluster/index.rst (hunk 1 above)
Comment (BuJianlin) [reply documents/readthedoc/docs/source/Solutions/tensorflow-serving-cluster/index.rst]: > SSL/TLS can be implemented either one-way TLS authentication or two-way TLS authentication (mutual TLS authentication).

Updated.

File: documents/readthedoc/docs/source/Solutions/tensorflow-serving-cluster/index.rst (hunk 2 above)
Comment (BuJianlin) [reply documents/readthedoc/docs/source/Solutions/tensorflow-serving-cluster/index.rst]: > two-way TLS authentication

Updated.

File: documents/readthedoc/docs/source/Solutions/tensorflow-serving-cluster/index.rst (hunk 3 above)
Comment (BuJianlin) [reply documents/readthedoc/docs/source/Solutions/tensorflow-serving-cluster/index.rst]: > one-way TLS authentication(client verifies server)

Updated.

File: documents/readthedoc/docs/source/Solutions/tensorflow-serving-cluster/index.rst (hunk 4 above)
Comment (BuJianlin) [reply documents/readthedoc/docs/source/Solutions/tensorflow-serving-cluster/index.rst]: > one-way TLS authentication::

Updated.

File: documents/readthedoc/docs/source/Solutions/tensorflow-serving-cluster/index.rst (hunk 5 above)
Comment (BuJianlin) [reply documents/readthedoc/docs/source/Solutions/tensorflow-serving-cluster/index.rst]: > two-way TLS authentication::

Updated.

**Review by yuanwu2017 — APPROVED** [review]

**Review by YangLiang3 — APPROVED** [review]

**Pull request merged**